packages/twitch-chat-client/src/ChatUser.ts

```diff
@@ -11,6 +11,15 @@ export default class ChatUser {
 		this._userData = userData ? new Map(userData) : new Map();
 	}
 
+	/** @private */
+	_parseBadgesLike(badgesLikeStr?: string): Map<string, string> {
```

Comment:
Nitpick: Please make it TS-private instead of JSDoc-private. The latter is only necessary when something is for internal use in another class, but shouldn't appear in the docs.